src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    /** List of tasks added by the user */
+    private static final List<Task> tasks = Storage.getData();
```

Comment:
Perhaps it would be better to rename tasks as TASKS since it is a final variable.